Student's grade.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE hmtl>
+<html>
+<head>
+<h1 id="myHeader"><em>Professor Steve's</em> student grade records</h1>
+<p class="bodytext">"These are the list of Professor Steve's student grades and courses which will be used to analyse and determine the number of student that takes his courses,<br> this analysis is in no way accurate, so pay no mind I just really want to fill up this paragraph.</br><br> It should be noted that Professor Steve may be like Ross Geler, student may not like him lol because well who likes a geeky professor that talks and blabs about Fossils and dinosaurs right?!"</p>
+<style>
+table, th, td{
+	border: 1px solid black;
+	  }
+	</style> 
+</head>
+<body>
+	<h2> Students, Courses and Grades </h2>
+<table style="width:50%">
+	<tr>
+		<th> Student's Name</th>
+		<th> Course Title</th>
+		<th> Grade</th>
+	</tr>
+	<tr>
+		<td>Rachel Green</td>
+		<td> Fashion Mgt</td>
+		<td> A3</td>
+	</tr>
+	<tr>
+		<td> Monica Geller</td>
+		<td> Culinary 403</td>
+		<td> A1</td>
+	</tr>
+	<tr>
+		<td> Pheobe Buffay</td>
+		<td> Introduction to Music</td>
+		<td> D</td>
+	</tr>
+	<tr>
+		<td> Ross Geler</td>
+		<td> Fossils </td>
+		<td>A</td>
+	</tr>
+	<tr>
+		<td> Chandler Bing</td>
+		<td> Transposter</td>
+		<td> B3</td>
+	</tr>
+	<tr> 
+		<td>Joey Tribianni</td>
+		<td> French</td>
+		<td>F9</td>
+	</tr>
+</table>
+<h3> An Unordered list of Student's Courses</h3>
+<ul>
+	<li> Fashion Mgt</li>
+	<li> Culinary 403</li>
+	<li> Introduction to Music</li>
+	<li>Fossils</li>
+	<li>Transposter</li>
+	<li>French</li>
+</ul>
+<h4>An Ordered list of Student's Courses</h4>
+<ol>
+	<li> Fashion Mgt</li>
+	<li> Culinary 403</li>
+	<li> Introduction to Music</li>
+	<li>Fossils</li>
+	<li>Transposter</li>
+	<li>French</li>
+	</ol>
+	<button> Male</button> 
```

Comment:
For the Gender, it is better to use radio buttons
Take a look at the example below, for future reference : 
https://www.w3schools.com/html/tryit.asp?filename=tryhtml_radio